packages/messaging/lib/windows.js

```diff
@@ -32,6 +32,14 @@ export class WindowsMessagingTransport {
     constructor (config, messagingContext) {
         this.messagingContext = messagingContext
         this.config = config
+        this.globals = {
+            window,
+            JSONparse: window.JSON.parse,
```

Comment:
We should solve this in the abstract soon: https://app.asana.com/0/0/1204392417715156/f